lib/data/datasource/remote/api_service.dart

```diff
@@ -22,4 +21,47 @@ class ApiService {
     );
     return response;
   }
+
+  Future<dio.Response> getSimilarRecipes(int id) async {
+    final response = await client.get(
+      '${ApiConstants.baseUrl}${ApiConstants.recipesEndpoint}$id/${ApiConstants.similarRecipesEndpoint}',
```

Comment:
You can set the baseUrl inside Dio client, and here you only need to use `${ApiConstants.recipesEndpoint}$id/${ApiConstants.similarRecipesEndpoint}'`

Reply:
Done